Yael/integration views docs (#10570)
DataDog/documentation: content/en/infrastructure/process.md

Hosts that are running the integration, but where the integration is not enabled, can be found in the **Hosts** tab of the integrations tile.

### Integration views 

{{< img src="infrastructure/process/integration_views.png" alt="Integration Views" >}}

Once a third-party software has been detected, Live Processes makes it quick and easy to analyze the performance of that software. 
1. To start, click on *Views* at the top right of the page to open a list of pre-set options, including Ngingx, Redis, and Kafka. 

> 2. Select a view to scope the page to just the processes running that software. 

Reviewer: ```suggestion
2. Select a view to scope the page to only the processes running that software. 
```

> 3. When inspecting a heavy process, shift to the *Integration Metrics* tab to analyze the health of the software on the underlying host. If you have already enabled the relevant Datadog integration, you'll be able to view all performance metrics collected via the integration to dinstinguish between a host-level and software-level issue. For intance, seeing correlated spikes in process CPU and MySQL query latency may indicate that an intensive operation, such as a full table scan, is delaying the execution of other MySQL queries relying onthe same underlying resources. 

Reviewer: ```suggestion
3. When inspecting a heavy process, shift to the *Integration Metrics* tab to analyze the health of the software on the underlying host. If you have already enabled the relevant Datadog integration, you'll be able to view all performance metrics collected from the integration to distinguish between a host-level and software-level issue. For instance, seeing correlated spikes in process CPU and MySQL query latency may indicate that an intensive operation, such as a full table scan, is delaying the execution of other MySQL queries relying on the same underlying resources. 
```

> You can customize integration views (e.g. when aggregating a query for Nginx processes by host) and other custom queries by clicking the *+Save* button at the top of the page. Doing so will save your query, table column selections, and visualization settings. Create saved views for quick access to the processes you care about without addition configuration, and to share process data with your teammates.  

Reviewer: ```suggestion
You can customize integration views (for example, when aggregating a query for Nginx processes by host) and other custom queries by clicking the *+Save* button at the top of the page. Doing so will save your query, table column selections, and visualization settings. Create saved views for quick access to the processes you care about without addition configuration, and to share process data with your teammates.  
```
